content/mqtt/index.textile

```diff
@@ -0,0 +1,95 @@
+* If using a token, set the username to the token, and leave the password blank
+
+For example, in NodeJS "mqtt package":https://www.npmjs.com/package/mqtt, you'd need to specify the following:
+
+bc[nodejs]. {
+  var options = { keepAlive: 30,
+    username: 'FIRST_HALF_OF_API_KEY',
+    password: 'SECOND_HALF_OF_API_KEY',
+    port: 8883
+  };
+  var client = mqtt.connect('mqtts:mqtt.ably.io', options);
+}
+
+This will connect using TLS through MQTT to Ably.
+
+h2(#decoding). Decoding MQTT communication
+
+Any data published through or received by the MQTT adapter will be binary encoded, due to MQTT being a binary protocol. This means that you'll need to interpret the message to get the original contents out. For example, if you tried to send a string with MQTT, with JavaScript you'd need to do the following:
+
+bc[nodejs]. {
+  var decoder = new TextDecoder();
+  var command = decoder.decode(message.data);
+}
+
+In the above example, @command@ will now contain the message in its original string form.
+
+h2(#use-cases). Recommended usage
+
+We recommend you use MQTT to interact with Ably when:
+* Using devices which do not have a native Ably client library, such as Arduino platforms, C/C++ applications, and so on
+* If bandwidth is limited and you want to keep network traffic to a minimum
+
+h2(#supported). Supported features
+
+Our MQTT adapter only supports features supported by both the MQTT protocol and the Ably platform. So:
+
+* It only supports MQTT 3.1.1 clients only. Connection attempts using earlier protocol versions will be rejected
+* Publishing supports QoS 0 or 1. Subscribing only supports QoS 0
+* Session resumption is supported within the usual "Ably time limit of two minutes":https://support.ably.io/solution/articles/3000044639-connection-state-recovery
+* It doesn’t support any MQTT features that aren’t normally supported by Ably, such as @WILL@ messages, the @RETAIN@ flag or "wildcard channel subscriptions":https://support.ably.io/solution/articles/3000063123-can-i-attach-and-subscribe-to-channels-using-wildcards
+* It doesn't support any Ably features that aren't supported by the MQTT protocol, such as presence, history and push notifications (though presence may be supported soon). Of course, you can always use the "Ably REST API":https://www.ably.io/documentation/rest-api in conjunction with the MQTT adapter to access features available over REST
+
+h2(#security). Security
+
+We recommend that you always make use of "token authentication":https://www.ably.io/documentation/general/authentication#token-authentication over "basic authentication":https://www.ably.io/documentation/general/authentication#basic-authentication when trying to connect from devices you may not trust. In addition, if you're using basic authentication you'll be required to use SSL to connect to our adapter to ensure the API key cannot be accessed by someone listening in.
```

Comment:
Perhaps link to the page that explains why Basic vs Token and how to choose the right auth scheme?